index.js

```diff
@@ -14,7 +22,18 @@ function hide(args) {
 // show elements
 function show(args) {
   args.forEach((arg) => {
-    arg.style.display = 'block';
+    arg.style.display = 'flex';
+  });
+}
+
+// blur elements
```

Comment:
- [ ] OPTIONAL
Kindly ensure that your code hosted on the repository contains **no** comments, so as to keep your base code cleaner.